### DISCUSSION TITLE
Sorting issue

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#370] [Q&A] [answered]
The sort isnt working as expected.
![image](https://user-images.githubusercontent.com/78496825/189676991-d1c824ae-6543-4ab6-8910-5c275577a5e6.png)

**Comment by imed-ghomari** [comment, marked as answer]
Better to file a bug request where you can specify the plugin version, and to track in the roadmap. I think discussions are kept for ideas and questions.

**Reply by DannyHatcher-Git** [reply]
Ok, I have not used github for this sort of thing before, so I am figuring it all out as I go 😁

**Reply by imed-ghomari** [reply]
No problem, you can find it in the issues section.
